templates/definition/charger/ocpp-elvi.yaml

```diff
@@ -5,5 +5,25 @@ products:
       generic: Elvi
 params:
   - preset: ocpp
+  - name: meter
```

Comment:
Ne, genau das wollte ich egtl. nicht. Ocpp teilt mir ja mit ob es einen Zähler gibt! D.h. bei den Boxen die nicht die Wunschwerte liefern kann `metervalues` immer in die Config und der Ladepunkt entscheidet ob es lohnt die `metervalues` auch zu aktivieren.

Reply:
Umso besser. Dann kann diese unnötige Logik ja hier einfach wieder raus.